app/src/main/java/edu/uc/hickmadc/customnotifications/dto/Schedule.kt

```diff
@@ -1,7 +1,5 @@
 package edu.uc.hickmadc.customnotifications.dto
 
-import java.util.*
-
-class Schedule (var original_occurence : Calendar, var interval : Int, var schedule_id:Int=0) {
```

Comment:
I see now.... I think when I formatted the code it removed this for some reason, or I did it on accident.

Reply:
Looks like it is missing a variable.